backend/src/main/java/gov/cdc/usds/simplereport/config/DataSourceConfiguration.java

```diff
@@ -0,0 +1,113 @@
+package gov.cdc.usds.simplereport.config;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import javax.sql.DataSource;
+import liquibase.integration.spring.SpringLiquibase;
+import org.springframework.boot.autoconfigure.liquibase.LiquibaseProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+@Configuration
+public class DataSourceConfiguration {
+
+  // SimpleReport data source
+
+  @Bean
+  @Primary
+  @ConfigurationProperties(prefix = "spring.datasource.simplereport")
+  public DataSource primaryDataSource(
+      HikariConfig dataSourceProperties, LiquibaseProperties dataSourceLiquibaseProperties) {
+
+    if (dataSourceLiquibaseProperties.getUser() != null) {
+      dataSourceProperties.setUsername(dataSourceLiquibaseProperties.getUser());
+    }
+    if (dataSourceLiquibaseProperties.getPassword() != null) {
+      dataSourceProperties.setPassword(dataSourceLiquibaseProperties.getPassword());
+    }
```

Comment:
If there's no override here, the "normal" datasource username/password will be used (`spring.datasource.simplereport.username`) instead of the Liquibase specific one. This is necessary if we want to use Liquibase as a different DB user than the app (e.g. `simple_report_migrations`).